docs/deployment-examples.md

```diff
@@ -0,0 +1,331 @@
+    purefa_info{array_name="ARRAY01",os="Purity//FA",system_id="12345678-abcd-abcd-abcd-abcdef123456",version="6.5.0"} 1
+    ```
+
+    We expect to return a single line displaying `purefa_info` returning the name, Purity OS, system ID and the Purity version.
+    You can remove the `-silent | grep ^purefa_info` from the command to see a list of all results.
+
+## Container - Tokens File - http with API tokens file embedded in exporter
+
+1. Create a tokens.yaml file to pass to the exporter.
+    ```console
+    $ cat /directorypath/tokens.yaml
+    # alias: of the array to be used in the exporter query
+    array01:
+      # FQDN or IP address of the array
+      address: 'array01.fqdn.com'
+      # API token of a user on the array
+      api_token: 'a1b2c3d4-e5f6-1234-5678-a1b2c3d4e5f6'
+
+    # Example of a second array
+    array02:
+      address: 'array02.fqdn.com'
+      api_token: 'a1b2c3d4-e5f6-1234-5678-a1b2c3d4e5f7'
+    ```
+
+2. **Run the container**
+    Run the container this time specifying a volume (in our case a single file) to pass the token.yaml file from the host server to the guest container.
+    The container expects to see the file here: `/etc/pure-fa-om-exporter/tokens.yaml`
+
+    ```console
+    $ docker run -d -p 9490:9490 --name pure-fa-om-exporter --volume /directorypath/tokens.yaml:/etc/pure-fa-om-exporter/tokens.yaml quay.io/purestorage/pure-fa-om-exporter:latest
+    ```
+3. **Check the container is running**
+    ```console
+    $ docker ps
+    CONTAINER ID   IMAGE                                            COMMAND                  CREATED         STATUS         PORTS                                       NAMES
+    0076ded8a073   quay.io/purestorage/pure-fa-om-exporter:latest   "/pure-fa-om-exporte…"   10 seconds ago   Up 10 seconds   0.0.0.0:9490->9490/tcp, :::9490->9490/tcp   pure-fa-om-exporter
+    ```
+
+4. **Test the exporter**
+    Use `curl` to test the exporter returns results. We don't need to pass the bearer (API) token for authorization as the exporter has a record of these which makes queries simpler.
+    ```console
+    $ curl -X GET 'http://localhost:9490/metrics/array?endpoint=array01 ' -silent | grep ^purefa_info
+    purefa_info{array_name="ARRAY01",os="Purity//FA",system_id="12345678-abcd-abcd-abcd-abcdef123456",version="6.5.0"} 1
+    ```
+
+## Container - TLS - https passing API token with query
+TLS with Docker image is currently not supported. Please use executable binary only for this feature.
+
+# Executable Binary Deployment
+## Binary Specific Prerequisites
+
+Deploying the binary requires [go](https://go.dev) to compile the code and running the binary in a Linux environment.
+1. **Install go**: Go is required: to compile the build into an executable binary
+
+    Clear instructions: can be found here: [https://go.dev/doc/install](https://go.dev/doc/install)
+
+2. **Install git**
+ 
+    ```console
+    $ yum install git
```

Comment:
this is operating system dependent, so maybe not give an example and just say install using your operating system package management tool, eg yum or apt